plane/source/docs/fixed-wing-faq.rst

```diff
@@ -228,26 +230,26 @@ sensor data. The techniques are:
 When no airspeed sensor is available some parameters are not used for
 some purposes:
 
--  the TRIM_ARSPD_CM parameter is not used as an airspeed target in
-   auto flight. Instead the TRIM_THROTTLE parameter is used as base
+-  the :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` parameter is not used as an airspeed target in
+   auto flight. Instead the :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter is used as base
    throttle, with extra throttle added/removed to retain the target
    altitude
--  the ARSPD_FBW_MIN and ARSPD_FBW_MAX parameters are not used for
+-  the :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` and :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` parameters are not used for
    airspeed limiting in TECS, but they are still used for the stall
    prevention code, using the synthetic airspeed value
 
 Why does my trim change when I change modes?
 --------------------------------------------
 
 Some people experience a problem where their roll or pitch trim changes
-when they change flight modes. So for example they are trimmed with
-level aileron in manual with the aircraft level and when they change to
-FBWA mode the ailerons move significantly off centre trim.
+when they change flight modes. So for example, in FBWA the plane is flying level, but
+when changed to MANUAL mode the plane is no longer in level trim.
 
-One likely cause of this is that you have a transmitter that has per
+One cause of this is that you have a transmitter that has per
 flight mode trims. The Taranis is a good example of this if you use its
-builtin flight mode controls. What happens is you setup the plane with
-correct trims in MANUAL by adjusting using the trip tabs in flight, but
-those trims don't get used when you change flight modes. You need to
-change your transmitter settings so that the stick inputs are the same
-in all flight modes.
+built in flight mode controls. You need to be sure  your transmitter trim settings 
+are the same, so that the stick inputs are the same in all flight modes and  match those
+you used for the RC calibration setup.
+
+Another, is that you are not using :ref:`SERVO_AUTO_TRIM<SERVO_AUTO_TRIM>` to automatically adjust the trim
```

Comment:
I haven't used servo_auto_trim before... What I have done is to use TX trims only for MANUAL mode, with trim set to zero for all other modes. Does auto_trim really eliminate the need for trimming in manual mode?  I assume you would at least have to fly in a stabilized mode first on any given day (with variations in temperature, etc.) for the trim to be set correctly.

Reply:
it works very well, as long as you want trim to be the attitude of level AHRS and at cruise adjusted throttle for stable altitude...STAB and MANUAL will fly just like FBWA/B/CRUISE..

I never touch trims...or worry about plane trim on first hand launch(I always AUTOTAKEOFF on maidens)....just cal at 1500us neutrals and use auto-trim...works great...